resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -29,17 +29,23 @@ private[spark] object Config extends Logging {
       .stringConf
       .createWithDefault("default")
 
+  val CONTAINER_IMAGE =
+    ConfigBuilder("spark.kubernetes.container.image")
+      .doc("Container image to use for Spark containers. Individual container types " +
+        "(e.g. driver or executor) can also be configured to use different images if desired, " +
+        "by setting the container-specific image name.")
```

Comment:
nit: `container-specific` => `container-type-specific`
and add `" eg. spark.kubernetes.driver.container.image"`?

Reply:
Why would I mention just one specific way of overriding this?

I also have half a mind to just remove this since this documentation is not visible anywhere...